website/projects/scimaps/src/app/shared/components/book-overview/book-overview.component.html

```diff
@@ -0,0 +1,15 @@
+<div class="overview-container" *ngIf="book">
```

Comment:
No need for overview/book prefixing all classes
Also no need to test if the book exists (it is a required input)